test/Document.tests.ts

```diff
@@ -1,4 +1,5 @@
 import * as slimdom from '../src/index';
+import { expectArity } from '../src/util/errorHelpers';
```

Comment:
Unused import?

Reply:
Thanks, this must have autocompleted in there by accident